src/Analyzers/Microsoft.Analyzers.Local/Microsoft.Analyzers.Local.csproj

```diff
@@ -30,10 +28,6 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" />
     <PackageReference Include="Microsoft.CodeAnalysis.Analyzers" PrivateAssets="all" IncludeAssets="runtime; build; native; contentfiles; analyzers; buildtransitive" />
-
-    <PackageReference Include="System.Text.Json" PrivateAssets="all" GeneratePathProperty="true" />
-    <None Include="$(OutputPath)\$(AssemblyName).dll" Pack="true" PackagePath="analyzers/dotnet/cs" Visible="false" />
-    <None Include="$(PkgSystem_Text_Json)\lib\netstandard2.0\*.dll" Pack="true" PackagePath="analyzers/dotnet/cs" Visible="false" />
```

Comment:
Doesn't this indicate the changes in main are an improvement over the 8.0 branch? Should those be retained?

Reply:
I'm honestly not sure about this one. @geeknoid any clues on this one?